build.gradle

```diff
@@ -16,8 +16,121 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+// TODO: this is work in progress, please follow FINERACT-1171
 buildscript {
+    ext {
+        jacocoVersion = '0.8.5'
+    }
+    repositories {
+        jcenter()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+
+    dependencies {
+        classpath 'com.bmuschko:gradle-cargo-plugin:2.7.1'
+        classpath 'org.zeroturnaround:gradle-jrebel-plugin:1.1.10'
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.3.4.RELEASE'
+        classpath 'gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin:0.15.0'
+        classpath 'org.apache.openjpa:openjpa:3.1.2' // when upgrading, also change OpenJPA version repeated below in dependencyManagement!
```

Comment:
```suggestion
        classpath 'org.apache.openjpa:openjpa:3.1.2' // when upgrading, also change OpenJPA version repeated in fineract-provider/build.gradle!
```

Reply:
Fixed and pushed. 

Suggestion for another time: I think this would be easier by defining a variable in buildscript ext section and use that in both places. Then we can get rid of these comments.